test/helpers/kubectl.go

```
@@ -67,7 +68,7 @@ func (kub *Kubectl) ExecPodCmd(namespace string, pod string, cmd string) (string
 	command := fmt.Sprintf("%s exec -n %s %s -- %s", kubectl, namespace, pod, cmd)
 	stdout := new(bytes.Buffer)
 	stderr := new(bytes.Buffer)
-	exit := kub.Execute(command, stdout, stderr)
+	exit, _ := kub.Execute(command, stdout, stderr)
```

Why are we ignoring the error?

---

In this case, we only need if the exit was success or not. The error message is not essential.(A lot of them are supposed to fail)

---

Still, logging the errors in debug mode may be useful.

---

The main problem to log things that didn't work cause a lot of noise, and opens tasks like https://github.com/cilium/cilium/issues/2221 

Have a lot of logs that says fail when it is supposed to fail creates a lot of confusion.